easybuild/framework/easyblock.py

```diff
@@ -1444,8 +1444,11 @@ def extensions_step(self, fetch=False):
         self.clean_up_fake_module(fake_mod_data)
 
     def package_step(self):
-        """Package software (e.g. into an RPM)."""
-        pass
+        """Prepare package software (e.g. into an RPM) with fpm."""
+
```

Comment:
please don't do that!

Explanation:
rpms are a delivery mechanism, not a publication mechanism (think scp or, tar for that matter!)

if you insist too much on a license check, at least make it a list for conditional exit.